File: NodeJs/FollowMeNode/app/models/blogItemPost.js
var mongoose = require("mongoose");

var Schema = mongoose.Schema;

var BlogItemPostSchema = new Schema({

	BlogItemId: {type: Schema.Types.ObjectId, ref: "BlogItem"},
    Url: String,
    Title: String,
    ShortDescription: String,
    ImageUrl: String,
    CreateDate: {type: Date, default: Date.now}

},{collection:"BlogItemPosts"});

module.exports = mongoose.model("BlogItemPost", BlogItemPostSchema);

File: AspNetCore/FollowMePlain/src/FollowMePlain/Data/BlogItemPost.cs
using System;
using MongoDB.Bson;

namespace FollowMePlain.Data {
    public class BlogItemPost {
        public ObjectId Id { get; set; }
        public int BlogItemId { get; set; }
        public string Url { get; set; }
        public string Title { get; set; }
        public string ShortDescription { get; set; }
        public string ImageUrl { get; set; }
        public DateTime? CreateDate { get; set; }
    }
}